roles/nv_gpu/tasks/install_nv.yml

```diff
@@ -46,34 +46,55 @@
   args:
     warn: false # don't warn about using sed here
 
-- name: "Find the GPU Operator OperatorHub InstallPlan"
-  # TODO: use 'oc get installplan -loperators.coreos.com/gpu-operator-certified.openshift-operators'
-  # when we get rid of OCP 4.5 support
-  command:
-    oc get Subscription/gpu-operator-certified
-       -n openshift-operators
-       -ojsonpath={@.status.installPlanRef.name}
-  register: gpu_operator_installplan_name
-  until: gpu_operator_installplan_name.stdout != ""
-  retries: 150
-  delay: 30
-
-- name: "Approve the GPU Operator OperatorHub InstallPlan"
-  command:
-    oc patch InstallPlan/{{ gpu_operator_installplan_name.stdout }}
-       -n openshift-operators
-       --type merge
-       --patch '{"spec":{"approved":true}}'
-
-- name: "Wait for the GPU Operator OperatorHub ClusterServiceVersion"
-  command:
-    oc get  ClusterServiceVersion/{{ gpu_operator_csv_name }}
-      -oname
-      -n openshift-operators
-  register: gpu_operator_wait_csv
-  until: gpu_operator_wait_csv.stdout != ""
-  retries: 20
-  delay: 30
+- block:
+  - name: Find the GPU Operator OperatorHub InstallPlan
+    # TODO: use 'oc get installplan -loperators.coreos.com/gpu-operator-certified.openshift-operators'
+    # when we get rid of OCP 4.5 support
+    command:
+      oc get Subscription/gpu-operator-certified
+         -n openshift-operators
+         -ojsonpath={@.status.installPlanRef.name}
+    register: gpu_operator_installplan_name
+    until: gpu_operator_installplan_name.stdout != ""
+    retries: 20
```

Comment:
less retires?

Reply:
yes, replacing `retries: 150` by `retries: 20`, we don't want to wait forever 
(I put 150 when I didn't know that the command was wrong, I should have fixed it with the previous patch!)

Reply:
Ok I trust you ;) , just checking